static/bidder-params/kobler.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Kobler Adapter Params",
+  "description": "A schema which validates params accepted by the Kobler adapter",
+  "type": "object",
+
+  "properties": {}
```

Comment:
Curious, how do map impressions to your system without any kind of identifier?

Reply:
Since we're a contextual platform, we identify placements from the combination of the article URL and the allowed sizes. Both of these are always present without having to add any other identifier.